RPIPs/RPIP-28.md

```diff
@@ -3,8 +3,8 @@ rpip: 28
 title: Deposits Under the Minimum
 description: Allow minipool deposits that include an appropriate RPL stake, even when under the "minimum" RPL threshold 
 author: Valdorff (@Valdorff)
-discussions-to: 
-status: Draft
+discussions-to: https://dao.rocketpool.net/t/allow-minipool-deposits-while-under-min-rpl/2100/9
```

Comment:
```suggestion
discussions-to: https://dao.rocketpool.net/t/allow-minipool-deposits-while-under-min-rpl/2100
```
Link to first post rather than latest.